scripts/publish-biblio.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+
+npx ecmarkup --verbose spec.html --write-biblio biblio/biblio.json /dev/null
```

Comment:
is perhaps this supposed to be > /dev/null?

Reply:
no, I missed it but `spec.html` does not exist. We need to change it to `spec/index.html`.

Reply:
Ugh, this should be `spec/index.html`. I forgot this project puts its files in a different place than ecma262.

(And no, just `/dev/null` is correct.)

Reply:
actually, maybe both?

Reply:
https://github.com/tc39/ecma402/pull/866